src/carousel/Carousel.js

```diff
@@ -43,8 +43,8 @@ export default class Carousel extends Component {
         autoplayDelay: PropTypes.number,
         autoplayInterval: PropTypes.number,
         callbackOffsetMargin: PropTypes.number,
-        containerCustomStyle: ViewPropTypes ? ViewPropTypes.style : View.propTypes.style,
-        contentContainerCustomStyle: ViewPropTypes ? ViewPropTypes.style : View.propTypes.style,
+        containerCustomStyle: ViewPropTypes ? ViewPropTypes.style : View.propTypes ? View.propTypes.style : () => null,
+        contentContainerCustomStyle: ViewPropTypes ? ViewPropTypes.style : View.propTypes ? View.propTypes.style : () => null,
```

Comment:
Same as above:

```
        contentContainerCustomStyle: ViewPropTypes ? ViewPropTypes.style : View.propTypes?.style ? View.propTypes.style : () => null,
```

Reply:
Sorry - terrible at using this github stuff!!!

You change did not fix the problem, still got red errors in chrome....  The following (just in the Carousel.js) did fix it as a slight mod to yours:

```
containerCustomStyle: ViewPropTypes ? ViewPropTypes.style : View.propTypes?.style ? View.propTypes.style : () => null,
contentContainerCustomStyle: ViewPropTypes ? ViewPropTypes.style : View.propTypes?.style ? View.propTypes.style : () => null,
```

Reply:
Hi @cyphire. Which device and browser are you testing it from? The fix was working fine on a Windows 10 Chrome browser. The only difference in your code is checking that `View.propTypes` contains a property named `style`. In fact, you your code would be more readable if you'd replace the ternary operator with an or operator:

```suggestion
        contentContainerCustomStyle: ViewPropTypes ? ViewPropTypes.style : View.propTypes?.style || () => null,
```

But again, in which case do you get a `View.propTypes` without the `style` property?